game/resource/English/ability/units/siltbreaker/tooltip_siltbreaker_boss_protection.txt

```diff
@@ -0,0 +1,9 @@
+//=============================================================================
+// SiltBreaker Boss Protection
+//=============================================================================
+
+"dota_tooltip_ability_siltbreaker_boss_protection"                     "SiltBreaker Boss Protection"
+"dota_tooltip_ability_siltbreaker_boss_protection_Description"         "Whenever the boss is stunned or silenced this triggers a automatic protection that purges him and make him silence and stun immune for %duration seconds"
+
+"dota_tooltip_modifier_siltbreaker_boss_protection"                    "{dota_tooltip_ability_siltbreaker_boss_protection}"
+"dota_tooltip_modifier_siltbreaker_boss_protection_Description"         "{dota_tooltip_ability_siltbreaker_boss_protection_Description}"
```

Comment:
This should just say "Cannot be stunned or silenced" or something along those lines.

Reply:
The problem is that this ability works like BB bristle back skill. It is a passive with cooldown and protection duration.

Reply:
The modifier description should just describe what the modifier itself does though; the HUD already displays that the modifier has a duration. Modifiers also generally don't have long descriptions. The duration also won't display properly if you just copy the ability description.